compiler/il/OMRNode.hpp

```diff
@@ -333,7 +333,7 @@ class OMR_EXTENSIBLE Node
       TR::ILOpCode opcode; opcode.setOpCodeValue(opvalue);
       TR_ASSERT(!opcode.isIf(), "use createif or createbranch on this node\n");
       TR_ASSERT(opvalue != TR::arraycopy, "use createArraycopy to create this node");
-      TR_ASSERT(opvalue != TR::tstart, "use createWithRoomForFive to create node: %s", opcode.getName());
+      //TR_ASSERT(opvalue != TR::tstart, "use createWithRoomForFive to create node: %s", opcode.getName());//since using create(3,a,b,c) instead, need to comment this for now
```

Comment:
Yeah that doesn't seem like it should be right with a superficial look at the code...